src/application.js

```diff
@@ -1063,7 +1100,11 @@ class XtkSchema extends XtkSchemaNode {
          * @type {string}
          */
         this.labelSingular = EntityAccessor.getAttributeAsString(xml, "labelSingular");
-
+        /**
+         * The id for translation of the singular label
+         * @type {string}
+         */
+        this.labelSingularLocalizationId = this.id.replace(":", "__") + "__@labelSingular"
```

Comment:
why this is not done inside xtkSchemaNode?

Reply:
Because `labelSingular` exists only in `XtkSchema`